mhcnames/normalization.py

```diff
@@ -64,6 +64,13 @@ def normalize_allele_name(raw_allele):
     parsed_alleles = parse_classi_or_classii_allele_name(raw_allele)
     species = parsed_alleles[0].species
     normalized_list = [species]
+    # Optionally omit the alpha allele
```

Comment:
worth saying why you might want to omit it, otherwise it'll be confusing since not in the spirit of "normalizing"